pkgs/applications/display-managers/lightdm/default.nix

```diff
@@ -78,6 +110,10 @@ stdenv.mkDerivation rec {
       --replace /bin/rm ${busybox}/bin/rm
   '';
 
+  postInstall = ''
+    rm -rf $out/share/apparmor.d $out/share/init
```

Comment:
These directories don't look like they exist?

Reply:
my mistake, it's at `etc`

```
result
├── bin
│   ├── dm-tool
│   └── lightdm
├── etc
│   ├── apparmor.d
│   │   ├── abstractions
│   │   │   ├── lightdm
│   │   │   └── lightdm_chromium-browser
│   │   └── lightdm-guest-session
│   ├── dbus-1
│   │   └── system.d
│   │       └── org.freedesktop.DisplayManager.conf
│   ├── init
│   │   └── lightdm.conf
│   ├── lightdm
│   │   ├── keys.conf
│   │   ├── lightdm.conf
│   │   └── users.conf
│   └── pam.d
│       ├── lightdm
│       ├── lightdm-autologin
│       └── lightdm-greeter
```

Reply:
corrected.